src/sentry/static/sentry/app/stores/guideStore.jsx

```diff
@@ -29,6 +26,18 @@ const GuideStore = Reflux.createStore({
     this.listenTo(GuideActions.unregisterAnchor, this.onUnregisterAnchor);
     this.listenTo(OrganizationsActions.setActive, this.onSetActiveOrganization);
     this.listenTo(OrganizationsActions.changeSlug, this.onChangeSlug);
+
+    window.addEventListener('hashchange', this.onHashChange, false);
```

Comment:
Let's follow the convention and put this in componentDidMount

Reply:
done